examples/seismic/acoustic/operators.py

```diff
@@ -144,10 +146,28 @@ def ForwardOperator(model, geometry, space_order=4,
 
     # Create interpolation expression for receivers
     rec_term = rec.interpolate(expr=u)
-
+    # Build operator equations
+    equations = eqn + src_term + rec_term
+
+    if factor:
```

Comment:
This needs to be wrapped into a utility function as it's duplicated below

Reply:
I've created a function to construct usnaps.